src/views/CashlinkManage.vue

```diff
@@ -108,55 +112,56 @@ export default class CashlinkManage extends Vue {
             return;
         }
 
+        let transactionToSend;
         if (storedCashlink) {
-            // cashlink already sent and stored
+            // Cashlink is typically already sent as the sending happens right after storing the Cashlink. However, it
+            // might be that the sending failed and the user reloaded the page, in which case we try sending again.
+            transactionToSend = network.getUnrelayedTransactions({
+                recipient: storedCashlink.address,
+                value: storedCashlink.value,
+            })[0];
+            this.isTxSent = !transactionToSend;
             this.retrievedCashlink = storedCashlink;
-            this.isTxSent = true;
-            this.isManagementRequest = !this.cashlink; // freshly created cashlink or management request?
         } else {
+            // Cashlink can not have been sent yet because whenever it gets sent, it also has been added to the store.
```

Comment:
Just a suggestion:

```suggestion
            // Cashlink can not have been sent yet because whenever it gets sent, it was also added to the store.
```